validator/rules_src/uuidv3.py

```diff
@@ -0,0 +1,42 @@
+from validator.rules_src import Rule
+from uuid import UUID
+
+
+class UUIDv3(Rule):
+    """
+    The field under validation must be a valid Universally Unique Identifier version 3
+
+    Examples:
+    >>> from validator import validate
+    >>> reqs = {'data' : 'a3bb189e-8bf9-3888-9912-ace4e6543002'}
+    >>> rule = {'data' : 'uuidv3'}
+    >>> validate(reqs, rule) # True
+    True
+
+    >>> from validator import validate
+    >>> reqs = {'data' : 'bba617b4-364b-4a0d-9e96-cb8a24ef1bec'}
+    >>> rule = {'data' : 'uuidv3'}
+    >>> validate(reqs, rule) # True
+    False
+    """
+
+    def __init__(self):
+        Rule.__init__(self)
+
+    def check(self, arg):
+        check_answer = True
```

Comment:
Please remove `check_anwer` variable and just return answer. It is easier for users to review your code

Reply:
done